package/common_init/overlay/etc/logrotate.conf

```diff
@@ -0,0 +1,25 @@
+# Directly copied from github.com/logrotate/logrotate/blob/master/examples/logrotate.conf
+
+# see "man logrotate" for details
+# rotate log files weekly
```

Comment:
Since most of this is commented out can we remove everything except the `include` line?